index.spec.js

```diff
@@ -0,0 +1,74 @@
+const request = require('supertest');
+
+const server = require('./api/server.js');
+
+
+describe('server', () => {
+  describe('GET /', () => {
+    
+    it('should return status code 200', async () => {
+      const res = await request(server).get('/');
+  
+      expect(res.status).toBe(200);
+    });
+
+    it('should return JSON', async () => {
+      const res = await request(server).get('/');
+      expect(res.type).toBe('application/json');
+    });
+
+    it('should return { message: "Server g2g" }', async  () => {
+      const res = await request(server).get('/');
+
+      expect(res.body).toEqual({ message: 'Server g2g' });
+    })
+  });
+  
+  describe('GET /games', () => {
+    it('returns the array of games', async () => {
+      const res = await request(server).get('/games');
+
+      expect(res.type).toBe('application/json');
+    });
+
+    it('returns status code 200', async () => {
+      const res = await request(server).get('/games');
+  
+      expect(res.status).toBe(200);
+    });
+
+    it('returns an array', async () => {
+      const res = await request(server).get('/games');
```

Comment:
Since you're doing this same request in each test in this describe (the one for GET /games), you could use a `beforeEach` instead of having the repetition:
```javascript
describe('GET /games', () => {
  let res;
  beforeEach(async () => {
    res = await request(server).get('/games');
  });

  it('returns the array of games', async () => {
     expect(res.type).toBe('application/json');
  });

  it('returns status code 200', async () => {
     expect(res.status).toBe(200);
  });

  it('returns an array', async () => {
    expect(Array.isArray(res.body)).toBe(true);
  });
```
This is equivalent since a new request will be made each go-round thanks to the `beforeEach`, and the response will be stored in the `res` variable each time.

Reply:
This sort of thing is more useful for repetitive GET tests over any other kind of test, but it can sometimes be used for other operations as well.